src/odb/src/zutil/util.cpp

```diff
@@ -47,8 +49,35 @@ using std::vector;
 RUDYCalculator::RUDYCalculator(dbBlock* block) : block_(block)
 {
   gridBlock_ = block_->getDieArea();
+  // from RUDY paper: "The wire width p is defined by the
+  //  average wire-to-wire pitch ~p and number of routing layers l: p = ~p/l.
```

Comment:
I didn't understand this comment with the line below. The current implementation isn't correct, then?

Reply:
> I didn't understand this comment with the line below. The current implementation isn't correct, then?

The current implementation is considering the width of the first layer. Although, the paper mentions this averaged value. I tried implementing it the way the paper mentions and the congestion values got too high. 

My idea is to test the current behavior with my modifications on multiple designs so we could further discuss what to do.

Reply:
Got it. Perhaps update this comment with a TODO message, something like:
`TODO: Match the wire width with the paper definition...`, completing with the definition you have there.